gbpservice/nfp/service_vendor_agents/vyos/agent/src/config_server/fw_module.py

```diff
@@ -1,50 +1,49 @@
-# One Convergence, Inc. CONFIDENTIAL
-# Copyright (c) 2012-2015, One Convergence, Inc., USA
-# All Rights Reserved.
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
 #
-# All information contained herein is, and remains the property of
-# One Convergence, Inc. and its suppliers, if any. The intellectual and
-# technical concepts contained herein are proprietary to One Convergence,
-# Inc. and its suppliers.
+#         http://www.apache.org/licenses/LICENSE-2.0
 #
-# Dissemination of this information or reproduction of this material is
-# strictly forbidden unless prior written permission is obtained from
-# One Convergence, Inc., USA
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
 
-#!/usr/bin/env python
+import ast
+import fw_constants
 import logging
-import json
 import netifaces
 import time
-import fw_constants
-import ast
+
+from oslo_serialization import jsonutils
+
+from execformat.executor import session
+from netifaces import AF_LINK
 from operations import configOpts
 from vyos_session import utils
-from netifaces import AF_INET, AF_INET6, AF_LINK, AF_PACKET, AF_BRIDGE
-from execformat.executor import session
 
 FWN = 'firewall name'
-# oc_fw_identifier = 'oc_fw'
 rule = 'rule'
 firewall_rules = {
-                    'protocol': '%s protocol %s',
-                    'source_ip_address': '%s source address %s',
-                    'destination_ip_address': '%s destination address %s',
-                    'source_port': '%s source port %s',
-                    'destination_port': '%s destination port %s'
-                }
+    'protocol': '%s protocol %s',
+    'source_ip_address': '%s source address %s',
+    'destination_ip_address': '%s destination address %s',
+    'source_port': '%s source port %s',
+    'destination_port': '%s destination port %s'
+}
 
 firewall_action = {'allow': 'accept', 'deny': 'drop'}
 
 logger = logging.getLogger(__name__)
 utils.init_logger(logger)
 
 
-class OCFWConfigClass(configOpts):
+class VyosFWConfigClass(configOpts):
 
     def __init__(self):
-        super(OCFWConfigClass, self).__init__()
-        self.oc_fw_identifier = 'oc_fw'
+        super(VyosFWConfigClass, self).__init__()
+        self.fw_identifier = 'fw'
```

Comment:
Looks like this class var is used as constants. Should we use it as constants instead of class variable ?